src/components/ai-input.tsx

```diff
@@ -18,28 +18,29 @@ interface AIInputProps {
 
 const MAX_ROWS = 10;
 
-export const AIInput = ( {
-	disabled,
-	input,
-	setInput,
-	handleSend,
-	handleKeyDown,
-	clearInput,
-	isAssistantThinking,
-}: AIInputProps ) => {
-	const inputRef = useRef< HTMLTextAreaElement >( null );
-
+const UnforwardedAIInput = (
+	{
+		disabled,
+		input,
+		setInput,
+		handleSend,
+		handleKeyDown,
+		clearInput,
+		isAssistantThinking,
+	}: AIInputProps,
+	inputRef: React.RefObject< HTMLTextAreaElement > | React.RefCallback< HTMLTextAreaElement > | null
+) => {
 	const [ isTyping, setIsTyping ] = useState( false );
 	const typingTimeout = useRef< NodeJS.Timeout >();
 
 	const { RiveComponent, inactiveInput, thinkingInput, typingInput, startStateMachine } =
 		useAiIcon();
 
 	useEffect( () => {
-		if ( ! disabled && inputRef.current ) {
-			inputRef.current.focus();
+		if ( ! disabled && inputRef && 'current' in inputRef && inputRef.current ) {
```

Comment:
This check and the declared `inputRef` type are [necessary](https://stackoverflow.com/a/66510585/378228) to satisfy the range of possible values when both referencing `inputRef.current` and forwarding the `inputRef` further.